2-mandatory/1-create-functions.js

```diff
@@ -24,7 +28,16 @@ Write a function that:
 - Removes any forward slashes (/) in the strings.
 - Makes the strings all lowercase.
 */
-function tidyUpString() {
+function tidyUpString(array) {
+  let tidyArray = [];
+  array.forEach((item) => {
+    const trimmedString = item.trim();
+    const removedSlash = trimmedString.replace("/", "");
+    const loweredCase = removedSlash.toLowerCase();
```

Comment:
you can chain these functions calls, you don't really need these local variables
`item.trim().replace("/", "").toLowerCase()`